css-brochure-form/styles.css

```diff
@@ -0,0 +1,64 @@
+h2 {
+  color: #ec5d57;
+  font-family: 'Brandon Grotesque', serif;
```

Comment:
The backup for this should actually be `sans-serif`. You can tell the difference between `serif` & `sans-serif` by looking at the text and seeing if they have those decorative lines.

![image](https://github.com/e-bales/c0523-code-solutions/assets/71413368/3b5fa8d0-3848-445d-ae8b-afaa26b8a29d)